### ISSUE TITLE
sendFile 

### ISSUE THREAD

**Issue by thelordapofis** [orkestral/venom#1723]
## Description

![image](https://user-images.githubusercontent.com/59453913/172701559-eead5a71-ef20-4321-8152-ba2579fd3d6e.png)

## Environment

- **Venom version(s):** [e.g. 4.1.0]

## Your Code

```
 messageAt = await client.sendFile(`${destinatario}@c.us`, filePath, fileName, "")
```

**Comment by singhamlagatari** [comment]
I'm found like this

**Comment by jsuceir** [comment]
up

**Comment by pedruhb** [comment]
Same problem here.

I tested with sendfile, sendimage, sendfilebase64, sendimagebase64.
Only works the sendimageasticker.

`            throw new Error('Evaluation failed: ' + helper_js_1.helper.getExceptionMessage(exceptionDetails));
                  ^

Error: Evaluation failed: TypeError: Cannot read properties of undefined (reading '0')
    at Object.u [as sendFile] (C:\Users\pedro\Documents\GitHub\SMSActivateWhatsAppBot\node_modules\venom-bot\dist\lib\wapi\wapi.js:1:25584)
    at ExecutionContext._evaluateInternal (C:\Users\pedro\Documents\GitHub\SMSActivateWhatsAppBot\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:221:19)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async ExecutionContext.evaluate (C:\Users\pedro\Documents\GitHub\SMSActivateWhatsAppBot\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:110:16)
    at async C:\Users\pedro\Documents\GitHub\SMSActivateWhatsAppBot\node_modules\venom-bot\dist\api\layers\sender.layer.js:528:28`

**Comment by arbisyarifudin** [comment]
Sama issue here.

**Comment by acognigni-evotecnia** [comment]
discussion and fix  already at #1718
